charts/opentelemetry-demo/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.27.0
```

Comment:
how about bump up to `0.28.0`?

Reply:
I've typically reserved minor upgrades for when the Demo itself would get bumped a minor version or we had a structural change to this chart itself